cleanlab/datalab/datalab.py

```diff
@@ -0,0 +1,632 @@
+        return self.issue_summary[row_mask].reset_index(drop=True)
+
+    def _add_issue_summary_to_report(self, summary: pd.DataFrame) -> str:
+        return (
+            "Here is a summary of the different kinds of issues found in the data:\n\n"
+            + summary.to_string(index=False)
+            + "\n\n"
+            + "(Note: A lower score indicates a more severe issue across all examples in the dataset.)\n\n\n"
+        )
+
+    def find_issues(
+        self,
+        *,
+        pred_probs: Optional[np.ndarray] = None,
+        features: Optional[npt.NDArray] = None,
+        knn_graph: Optional[csr_matrix] = None,
+        model=None,  # sklearn.Estimator compatible object  # noqa: F821
+        issue_types: Optional[Dict[str, Any]] = None,
+    ) -> None:
+        """
+        Checks the dataset for all sorts of common issues in real-world data (in both labels and feature values).
+
+        You can use Datalab to find issues in your data, utilizing *any* model you have already trained.
+        This method only interacts with your model via its predictions or embeddings (and other functions thereof).
+        The more of these inputs you provide, the more types of issues Datalab can detect in your dataset/labels.
+        If you provide a subset of these inputs, Datalab will output what insights it can based on the limited information from your model.
+
+        Note
+        ----
+        The issues are saved in the ``self.issues`` attribute, but are not returned.
+
+        Parameters
+        ----------
+        pred_probs :
+            Out-of-sample predicted class probabilities made by the model for every example in the dataset.
+            To best detect label issues, provide this input obtained from the most accurate model you can produce.
+
+        features :
+            Feature embeddings (vector representations) of every example in the dataset.
+
+        knn_graph :
+            Sparse matrix representing similarities between examples in the dataset in a K nearest neighbor graph.
+            If both `knn_graph` and `features` are provided, the `knn_graph` will take precendence.
+            If `knn_graph` is not provided, it is constructed based on the provided `features`.
+            If neither `knn_graph` nor `features` are provided, certain issue types like (near) duplicates will not be considered.
+
+        model :
+            sklearn-compatible model trained via cross-validation to compute out-of-sample
+            predicted probabilities for each example. Only considered if `pred_probs` was not provided.
+
+            WARNING
+            -------
+            This is not yet implemented.
+            
+        issue_types :
+            Collection specifying which types of issues to consider in audit and any non-default parameter settings to use.
+            If unspecified, a default set of issue types and recommended parameter settings is considered.
```

Comment:
@elisno TODO: link to the method which user can call to see the default `issue_types` dict

Reply:
Added:

https://github.com/cleanlab/cleanlab/pull/614/commits/31d86f4001578866ba1f2d9d1142d7ba8dcefccf#diff-129eb9ae1954a1b587ce56ff6acbd2f2d348b768cd20fa19b97e3e5a530bd693R88-R97

Reply:
I'd explicitly link to the `list_default_issue_types()` method in this docstring.